CHANGES.md

```diff
@@ -1,3 +1,20 @@
-### 1.0.0
+# Changelog
+
+## Unreleased
+
+## 1.0.1 (2019-11-28)
+
+### Added
+
+- Provide a better CLI interface for the benchmarks (#130, #133)
+
+### Fixed
+
+- Do not allocate 64k bytes buffers on the thread stack (#132)
```

Comment:
```suggestion
- Do not allocate 64k-byte buffers on the thread stack (#132)
```

Reply:
Can you specify the error mode (eg segfault on old musl)?

Reply:
Fixed.